__tests__/api.test.js

```diff
@@ -43,6 +43,45 @@ describe('/api/topics', () => {
         })
     })
 })
+describe('/api/articles', () => {
+    describe('GET', () => {
+        it('returns an array of articles from the db', () => {
+            return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({body: {articles}}) => {
+                articles.forEach((article) => {
+                    expect(article).toEqual({
+                        author: expect.any(String),
+                        title: expect.any(String),
+                        article_id: expect.any(Number),
+                        topic: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        article_img_url: expect.any(String),
+                        comment_count: expect.any(Number)
+                    })
+                })
+            })
+        })
+        it('returns all of the articles from the db', () => {
+            return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({body: {articles}}) => {
+                expect(articles.length).toBe(13)
```

Comment:
Please merge the forEach and length check tests, as one request may be working and the other not.